doc/whats-new.md

```diff
@@ -1,413 +1,340 @@
-version of Python with a different pickle protocol, like Python 3.8.
+`oq plot` has been improved for disaggregation outputs, making it possible
+to plot all kinds of outputs and also to compare disaggregation plots.
+We also added a command `oq plot vs30?` to plot the vs30 field of the
+site collection.
+
+We added a command `oq nrml_to` to convert source models into CSV and/or
+geopackage format. This is essential in order to plot source models with
+QGIS and it is meant to replace the command `oq to_shapefile` which has
+many limitations.
+
+We added a command `oq recompute_losses <calc_id> <aggregate_by>`: this is
+useful if you want to aggregate an ebrisk calculation with respect to
+different tags and you do not want to repeat the whole calculation.
+
+We fixed a 32 bit/64 bit bug in `oq prepare_site_model` affecting the case
+when the file `sites.csv` is the same as the `vs30.csv` file.
+
+The semantic of the command ``oq engine --run --reuse_hazard`` has
+been changed: before it was finding an existing hazard calculation (if
+any) from the checksum of the input files and using it, now it is just
+reusing the cached source model, if present. The change was necessary
+since the checksum of the calculation often changes between version;
+moreover reusing the GMFs generated with a previous version of the
+engine is often inconsistent, i.e. it will not give the same results
+since the details of the seed generation may change.
+
+The `oq engine` command has been enhanced and it is now possible to run
+multiple jobs in parallel as follows:
+```
+oq engine --multi ---run job1.ini job2.ini ... jobN.ini
+```
+This is useful if you have many small calculations to run
+(for instance many scenarios). Without the ``--many`` flag the
+calculations will be run sequentially, as in previous versions of the engine.
+
+# WebUI/WebAPI/QGIS plugin
+
+When running a classical calculation with the parameter `poes`
+specified, the engine now produces hazard maps in .png format that are
+visible through the WebUI by clicking the button on the bottom left
+side called "Show hazard map".  This is meant for debugging purposes
+only, the recommended way to visualize the hazard maps is still to use
+the QGIS plugin.
+
+There is a new API for storing JSON information inside .npz files, which
+is being used to store metadata. The QGIS plugin has been adapted to take
+advantage of the new API and the command `oq extract` has been changed
+to save in .npz format and not in .hdf5 format.
+
+We changed the call to `/extract/events` to return only the relevant events,
+i.e. the events producing nonzero GMFs, in case a `minimum_intensity` is
+specified.
+
+The `/extract/sitecol` API has been extended so that it is possible to
+extract selected columns of the site collection, for instance
+`/extract/sitecol?param=custom_site_id`, if a `custom_site_id` is defined.
+
+# Packaging
+  
+Python 3.8 is now fully supported and using Python 3.6 is deprecated
```

Comment:
Should we update https://github.com/gem/oq-engine/blob/master/setup.py#L112-L114 to include Python 3.8?

Reply:
GDAL wheels are missing from the requirements-extra-py38 files..

Reply:
There are missing from extra because they are not extra anymore ;-) They are in the requirements-py38.txt

Reply:
> Should we update https://github.com/gem/oq-engine/blob/master/setup.py#L112-L114 to include Python 3.8?

No, because Python 3.6 is still working, there is no point in forbidding its usage.

Reply:
We can add Python 3.8 though 😉